Development/cpprest/json_utils.cpp

```diff
@@ -74,17 +75,25 @@ namespace web
         // returns true if the value has at least one field matching the key path
         // if any arrays are encountered on the key path, results is an array, otherwise it's a non-array value
         bool extract(const web::json::object& object, web::json::value& results, const utility::string_t& key_path)
+        {
+            std::vector<utility::string_t> key_path_;
+            boost::algorithm::split(key_path_, key_path, [](utility::char_t c) { return U('.') == c; });
```

Comment:
I think we try to avoid `U` preprocessor macro in the library code? Use `_XPLATSTR` as below.

Reply:
Okay, will change to use `_XPLATSTR` instead of `U`.

FYI `U` is already used in the preprocess function.
            

>utility::string_t preprocess(const utility::string_t& value)
>{
>    // regex pattern matches JSON strings, or single or multi-line comments
>    // only strings are captured
>    static const utility::regex_t string_or_comment(U(R"-regex-(("[^"\\]*(?:\\.[^"\\]*)*")|(?:\/\/[^\r\n]+)|(?:\/\*[\s\S]*?\*\/))-regex-"));
>    // format pattern uses the first capture group to copy strings into the output
>    // having inserted a single space to ensure tokens are not coalesced
>    return bst::regex_replace(value, string_or_comment, U(" $1"));
>}

Reply:
> FYI `U` is already used in the preprocess function.

You're right. I think the intention may have been that anything we might conceivably have wanted to push back to Microsoft/cpprestsdk should use `_XPLATSTR` so that the code could be built with `U` disabled. So that probably means anything in the _Development/cpprest_ (or _pplx_) directories... However, there are obviously a few files where I failed to follow my own rule already, including the instance you pointed out! Moreover, since cpprestsdk is in maintenance mode, it's not going to happen, even for the least controversial additions. For now, why not just fix the rest of _this_ file so it uses `_XPLATSTR` consistently, and leave it at that. Thanks. :-)

Reply:
okay